docs/docs/modules/indexes/document_loaders/examples/file_loaders/docx.md

```diff
@@ -0,0 +1,25 @@
+---
+hide_table_of_contents: true
+---
+
+# Docx files
+
+This example goes over how to load data from docx files.
+
+# Setup
+
+```bash npm2yarn
+npm install mammoth
+```
+
+# Usage
+
+```typescript
+import { DocxLoader } from "langchain/document_loaders";
+
+const loader = new PDFLoader(
```

Comment:
hey @aptlin , it should be DocxLoader rigth?

Reply:
Right! Fixed in [5c00d4f](https://github.com/hwchase17/langchainjs/pull/556/commits/5c00d4f3ed2d82fbdf6a679bdc827cc7d3a67ad6)